### DISCUSSION TITLE
Makes and Models

### DISCUSSION THREAD

**Discussion by wesshaw** [Budibase/budibase#4785] [Help] [answered]
Suppose I have a table of makes and models:
| Make      | Model |
| ----------- | ----------- |
| Ford      | Focus       |
| Toyota   | Prius        |
| Ford      | Mustang       |
| Toyota   | Yaris        |
| Nissan   | Pathfinder     |

Is it possible within a form to have two options pickers, one for the Make and one for the Model where the Model is filtered based on the currently selected Make?  This is how I currently am trying to implement this problem so that I can have the Options manageable by an Admin instead of needing to be "developed" into the platform.  I've tried custom queries, and integrating that with separate Data Providers, but I may be overcomplicating things.  Any suggestions for a better route?

**Comment by aelias36** [comment]
I would suggest having two tables: a make table and a model table.

The make table only has makes.

The model table has one field which references a make (in the make table), and another field for the model.

Now your make picker can select based on the make table, and the model picker can pick based on the filtered model table.

**Comment by wesshaw** [comment, marked as answer]
I found something similar that helped: https://github.com/Budibase/budibase/discussions/4519.  Also, thanks for the suggestion @aelias36 .
